vg/vggio/vggio_test.go

```diff
@@ -24,6 +26,45 @@ import (
 
 const deltaGio = 0.05 // empirical value from experimentation.
 
+func init() {
```

Comment:
This is to check that vggio works without timing out correct? If that's the case, I'd prefer that it be in a real test; it can be made a subtest that the other tests are also in to ensure that it runs before them.

Is there any clue why this fails on GHA?

Reply:
my working hypothesis was first that Xvfb needed some time to boot up correctly (so I added a `sleep 30s` in the corresponding yaml file), but that wasn't it.

my current working hypothesis is that Xvfb sets up the headless display "on demand" and there might be some race or delay between its availability and the first time it is requested.

thus the `init` function which would need to be executed *before* all other `TestXYZ` funcs.

I guess one could simplify that `init` func and just execute `test` twice and return.

Reply:
Does this work? It has less boiler plate.
```
func init() {
	if runtime.GOOS != "linux" {
		return
	}

	const (
		w   = 20 * vg.Centimeter
		h   = 15 * vg.Centimeter
		dpi = 96
	)
	gtx := layout.Context{
		Ops: new(op.Ops),
		Constraints: layout.Exact(image.Pt(
			int(w.Dots(dpi)),
			int(h.Dots(dpi)),
		)),
	}

	for try := 0; try < 2; try++ {
		_, err := New(gtx, w, h, UseDPI(dpi)).Screenshot()
		if err == nil {
			return
		}
	}
	panic(fmt.Errorf("vg/vggio_test: could not setup headless display: %+v", err))
}
```

Reply:
it does (with a minimal fix for the dangling `err` variable)